source/shared/net/guides/run-node.rst

```diff
@@ -34,39 +34,10 @@ Before running a Concordium node you will need to
 
 2. Download and extract the :ref:`full suite for running a node on Linux<concordium-docker-package-download>`.
 
-Upgrading from version 1.1.3 to 3.0.0
-=====================================
-
-To upgrade do the following steps after downloading the new version.
-
-1. Run the ``concordium-node-stop`` executable from the unzipped archive and
-   wait for it to terminate. This might take a few seconds.
-
-2. Start the node again by running the ``concordium-node`` executable from the
-   unzipped archive. If your node is already caught up consider using
-   ``--no-block-state-import`` to avoid downloading data you already have.
-
-.. note::
-   If your node does not show on the network dashboard after upgrading to 3.0.0,
-   it could be that the node was started by the old version of the runner.
-   The old version of the software does not configure the docker container correctly, and this
-   misconfiguration can persist even if you then use the new version.
-   To fix this:
-
-   #. Ensure that you have downloaded and extracted the :ref:`latest version of the software<concordium-docker-package-download>`.
-
-   #. Check the version by running ``concordium-node --help``. It should show 3.0.0.
-
-   #. Stop any running node with ``concordium-node-stop``.
-
-   #. Remove the misconfigured docker container with ``docker rm concordium-client``.
-
-   #. Start the node again with ``concordium-node --no-block-state-import``.
-
 .. _running-a-node:
 
-Running a node
-==============
+Running/upgrading a node
```

Comment:
It would probably be useful to mention that to upgrade you have to restart the node (and that is the only thing needed, the tool will automatically check for newest version).